guides/source/active_record_basics.md

```diff
@@ -71,107 +70,153 @@ Convention over Configuration in Active Record
 When writing applications using other programming languages or frameworks, it
 may be necessary to write a lot of configuration code. This is particularly true
 for ORM frameworks in general. However, if you follow the conventions adopted by
-Rails, you'll need to write very little configuration (in some cases no
-configuration at all) when creating Active Record models. The idea is that if
-you configure your applications in the very same way most of the time then this
-should be the default way. Thus, explicit configuration would be needed
-only in those cases where you can't follow the standard convention.
+Rails, you'll write very little to no configuration code when creating Active Record models.
+
+Rails adopts the idea that if you configure your applications in the same way
+most of the time, then that way should be the default. Explicit configuration
+should be needed only in those cases where you can't follow the convention.
+
+To take advantage of convention over configuration in Active Record, there are
+some naming and schema conventions to follow. And in case you need to, it is possible to [override naming conventions](#overriding-the-naming-conventions).
 
 ### Naming Conventions
 
-By default, Active Record uses some naming conventions to find out how the
-mapping between models and database tables should be created. Rails will
-pluralize your class names to find the respective database table. So, for
-a class `Book`, you should have a database table called **books**. The Rails
-pluralization mechanisms are very powerful, being capable of pluralizing (and
-singularizing) both regular and irregular words. When using class names composed
-of two or more words, the model class name should follow the Ruby conventions,
-using the CamelCase form, while the table name must use the snake_case form. Examples:
+Active Record uses this naming convention to map between models (represented by
+Ruby objects) and database tables:
+
+Rails will pluralize your model's class names to find the respective database
+table. For example, a class named `Book` maps to a database table named
+**books**. The Rails pluralization mechanisms are very powerful and capable of
+pluralizing (and singularizing) both regular and irregular words. This uses the
+[Active Support](active_support_core_extensions.html#pluralize) [pluralize
+method](https://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-pluralize),
+if you're curious.
 
-* Model Class - Singular with the first letter of each word capitalized (e.g., `BookClub`).
-* Database Table - Plural with underscores separating words (e.g., `book_clubs`).
+For class names composed of two or more words, the model class name will follow
+the Ruby conventions of using a CamelCase name. The database table name, in that
```

Comment:
This should be PascalCase instead of camelCase.

Reply:
As CamelCase is the more common term and used in Rails since the beginning, I think it's fine to keep it.
Rails even has a method for it:
https://edgeapi.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-camelize

Reply:
Yes, it's common, but isn't that definition incorrect? If you ask someone to think of camel case, the first thought is `myVariable`, not `MyVariable`. That's pascal case to my understanding.

Reply:
Previous PR's to change it to PascalCase have been rejected: https://github.com/rails/rails/pull/36554 https://github.com/rails/rails/pull/42068

Reply:
Thanks for pointing this out @brunoprietog. And thanks for that `camelize` method api link and the history of these terms around Rails @p8.

As per that api doc, a distinction is made between UppderCamelCase and lowerCamelCase, and we will use that term in the Active Record guide.